internals/plan/plan_test.go

```diff
@@ -81,9 +82,10 @@ type planTest struct {
 	stop    map[string][]string
 }
 
-var planTests = []planTest{{
-	summary: "Relatively simple layer with override on top",
-	input: []string{`
+var planTests = []planTest{
+	{
+		summary: "Relatively simple layer with override on top",
+		input: []string{`
```

Comment:
Nothing should have changed here. Please undo these formatting changes and revert to the way things were originally.